staging/src/k8s.io/code-generator/cmd/informer-gen/generators/factory.go

```diff
@@ -106,8 +110,15 @@ type sharedInformerFactory struct {
 
 // NewSharedInformerFactory constructs a new instance of sharedInformerFactory
 func NewSharedInformerFactory(client {{.clientSetInterface|raw}}, defaultResync {{.timeDuration|raw}}) SharedInformerFactory {
+  return NewFilteredSharedInformerFactory(client, defaultResync, {{.namespaceAll|raw}}, nil)
+}
+
+// NewFilteredSharedInformerFactory constructs a new instance of sharedInformerFactory
+func NewFilteredSharedInformerFactory(client {{.clientSetInterface|raw}}, defaultResync {{.timeDuration|raw}}, namespace string, tweakListOptions {{.interfacesTweakListOptionsFunc|raw}}) SharedInformerFactory {
   return &sharedInformerFactory{
-		client: client,
+	client:           client,
+    namespace:        namespace,
```

Comment:
indention?

Reply:
Fixed - mixture between spaces and tabs confused my IDE!